testing/mopidy/APKBUILD

```diff
@@ -14,33 +14,60 @@ depends="python py-pykka py-tornado gst-plugins-base gst-plugins-good
 	py-six
 	"
 depends_dev=""
-makedepends="python-dev"
-install=""
-subpackages=""
+makedepends="python-dev py2-sphinx"
+install="$pkgname.pre-install"
+subpackages="$pkgname-doc"
 source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz
 	mopidy.conf
+	logging.conf
+	mopidy.initd
 	"
 
 _builddir="$srcdir"/$_pkgname-$pkgver
-prepare() {
-	default_prepare
-}
+
+pkgusers="mopidy"
+pkggroups="audio"
 
 build() {
 	cd "$_builddir"
 	python setup.py build
+
+	make -C docs SPHINXBUILD=sphinx-build-2 man
+	make -C docs SPHINXBUILD=sphinx-build-2 html
 }
 
 package() {
 	cd "$_builddir"
 	python setup.py install --prefix=/usr --root="$pkgdir"
-	install -Dm640 "${srcdir}/mopidy.conf" "${pkgdir}/etc/mopidy/mopidy.conf"
-	install -dm755 "${pkgdir}/etc/mopidy/extensions.d"
-	install -dm755 "${pkgdir}/var/cache/mopidy"
-	install -dm755 "${pkgdir}/var/log/mopidy"
-	install -dm755 "${pkgdir}/var/lib/mopidy/local"
-	install -dm755 "${pkgdir}/var/lib/mopidy/media"
-	install -dm755 "${pkgdir}/var/lib/mopidy/playlists"
+	install -Dm755 extra/mopidyctl/mopidyctl "${pkgdir}/usr/bin/mopidyctl"
+
+	install -dm755 "${pkgdir}/usr/share/doc/mopidy"
+	cp -r docs/_build/html "${pkgdir}/usr/share/doc/mopidy"
+
+	install -dm755 "${pkgdir}/usr/share/man/man1/"
+	gzip -c docs/_build/man/mopidy.1 > "${pkgdir}/usr/share/man/man1/mopidy.1.gz"
+
+	install -dm755 "${pkgdir}/usr/share/man/man8/"
+	gzip -c extra/mopidyctl/mopidyctl.8 > "${pkgdir}/usr/share/man/man8/mopidyctl.8.gz"
+
+	install -Dm644 "${srcdir}/mopidy.conf" \
+		"${pkgdir}/etc/mopidy/mopidy.conf"
+	install -Dm644 "${srcdir}/logging.conf" \
+		"${pkgdir}/etc/mopidy/logging.conf"
+
+	install -Dm755 "$srcdir/$pkgname.initd" \
+		"$pkgdir/etc/init.d/$pkgname"
+
+	install -Dm644 LICENSE "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
+
+	install -dm755 -o mopidy -g audio "${pkgdir}/etc/mopidy/extensions.d"
+	install -dm755 -o mopidy -g audio "${pkgdir}/var/cache/mopidy"
+	install -dm755 -o mopidy -g audio "${pkgdir}/var/log/mopidy"
+	install -dm755 -o mopidy -g audio "${pkgdir}/var/lib/mopidy/local"
+	install -dm755 -o mopidy -g audio "${pkgdir}/var/lib/mopidy/media"
+	install -dm755 -o mopidy -g audio "${pkgdir}/var/lib/mopidy/playlists"
+
+	ls -l "${pkgdir}/var/lib/mopidy"
```

Comment:
This is debug info?

Reply:
Yeah I pushed what I had as you asked for the scripts. Will be removed before merging